src/Eto/Eto.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.0</TargetFramework>
+	<GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DefineConstants>TRACE;DEBUG;PCL</DefineConstants>
```

Comment:
`<DefineConstants>TRACE;PCL</DefineConstants>`
I think you can move this to main `<PropertyGroup>` since Debug const is added by default on Debug build (if newly defined ones doesn't override.
And by moving both (DefineConstants and OutputPath) you could get rid of these PropertyGroups with conditions

Reply:
Hm, I'll try that, thanks!